_sass/_uswds-theme-overrides.scss

```diff
@@ -92,14 +92,14 @@ $c-dhs-green-90: #040602;
 
 // Primary colors
 $theme-color-primary-family: "blue-cool";
-$theme-color-primary-lightest: $c-dhs-blue-10;
-$theme-color-primary-lighter: $c-dhs-blue-20;
-$theme-color-primary-light: $c-dhs-blue-40;
-$theme-color-primary: $c-dhs-blue;
-$theme-color-primary-vivid: "blue-60v";
-$theme-color-primary-dark: $c-dhs-blue-60;
-$theme-color-primary-darker: $c-dhs-blue-70;
-$theme-color-primary-darkest: $c-dhs-blue-80;
+$theme-color-primary-lightest: $c-dhs-light-blue-10;
+$theme-color-primary-lighter: $c-dhs-light-blue-20;
+$theme-color-primary-light: $c-dhs-light-blue-40;
+$theme-color-primary: $c-dhs-light-blue;
+$theme-color-primary-vivid: "cyan-50v";
+$theme-color-primary-dark: $c-dhs-light-blue-60;
+$theme-color-primary-darker: $c-dhs-light-blue-70;
+$theme-color-primary-darkest: $c-dhs-light-blue-80;
```

Comment:
1. Now that we have global components (header/footer/identifier) with the right content we can start refining our USWDS theme overrides. The default primary color was change to the light-blue variation based on the colors on the homepage.

If you wish to switch back to the darker blue, you can easily override here.